sys/include/crypto/ed25519.h

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (C) 2015 Jan Wagner <mail@jwagner.eu>
+ * Copyright (C) 2014 Daniel Beer <dlbeer@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_crypto
+ * @{
+ *
+ * @file
+ * @brief       Headers for the implementation of the Edwards-curve equivalent of Curve25519 
```

Comment:
Please remove the trailing whitespace on this line and all other lines (as indicated by
https://travis-ci.org/RIOT-OS/RIOT/jobs/55205304 )